src/components/Navigation.tsx

```diff
@@ -32,9 +32,11 @@ export const Navigation = () => {
 													item.current
 														? 'bg-gray-900 text-white'
 														: 'text-white hover:bg-green/80 hover:text-purple',
+													item.external ? 'underline' : '',
```

Comment:
why only external link underline
![image](https://github.com/user-attachments/assets/915b86d6-7f7a-4c97-a6e2-4af3bc05fa84)

Reply:
Summit is the only external link and move to other website. What do you think so it it? Or maybe add a external icon next to it?

Reply:
I think external icon is better idea